src/test/java/org/sagebionetworks/web/client/SynapseJavascriptClientTest.java

```diff
@@ -138,7 +138,7 @@
 import org.sagebionetworks.web.shared.exceptions.UnauthorizedException;
 import org.sagebionetworks.web.shared.exceptions.UnknownErrorException;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.Silent.class)
```

Comment:
Without `Silent`, mockito fails tests that have mocks that are not called. Which is great, but a ton of work to fix.